src/components/profile/ProfileDirective.js

```diff
@@ -15,6 +15,7 @@ goog.require('ga_profile_service');
           templateUrl: 'components/profile/partials/profile.html',
           scope: {
             feature: '=gaProfile',
+            map: '=gaProfileMap',
```

Comment:
Yes probably better. I don't remenber why I put the map stuff inside the directive.